components/tls/tls-mpc/tests/test.rs

```diff
@@ -291,26 +291,53 @@ async fn test() {
 
     let conn_task = tokio::spawn(conn_fut);
 
+    let msg = concat!(
```

Comment:
I noticed this test is currently named 'test'. It would be really helpful to have a more descriptive name for better clarity 📛